pkg/virt-controller/services/template_test.go

```diff
@@ -389,6 +389,33 @@ var _ = Describe("Template", func() {
 			})
 		})
 
+		Context("without debug log annotation", func() {
+			It("should NOT add the corresponding environment variable", func() {
+				vmi := v1.VirtualMachineInstance{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      "testvmi",
+						Namespace: "default",
+						UID:       "1234",
+						Labels: map[string]string{
+							debugLogs: "false",
+						},
+					},
+				}
+
+				pod, err := svc.RenderLaunchManifest(&vmi)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(len(pod.Spec.Containers)).To(Equal(1))
+				debugLogsValue := ""
+				for _, ev := range pod.Spec.Containers[0].Env {
+					if ev.Name == ENV_VAR_LIBVIRT_DEBUG_LOGS {
+						debugLogsValue = ev.Value
+						break
+					}
+				}
+				Expect(debugLogsValue).To(Or(Equal(""), Equal("0")))
```

Comment:
@iholder-redhat Do we know in which case it will be equal to ```""``` and in which case it is ```"0"```?

Reply:
With current implementation the environment variable is defined to "1" if debug logs are enabled and not defined at all if debug logs are disabled.

I thought that this is an implementation details that might change in the future (e.g. the environment variable can be defined to "0", indicating to disable debug logs). Do we want the test to fail in such case?

Reply:
@iholder-redhat I think that if ```""``` and ```"0"``` are expected in  the real world scenrios, they deserve a separate test case per each. You can use ```table.DescribeTable``` for reducing code reuse.

Reply:
Didn't know table.DescribeTable.. Very cool!!